frontend/src/components/ide/AIAssistant/components/SettingsPanel.jsx

```
@@ -110,6 +124,119 @@ export function SettingsPanel({ settings, onSave }) {
+                        />
+                        <p className="text-xs text-muted-foreground mt-1">
+                            Максимальное количество токенов в ответе (256-32768)
+                        </p>
+                    </div>
+
+                    <div>
+                        <label className="text-sm font-medium mb-1 block">Системный промпт</label>
+                        <textarea
+                            value={systemPrompt}
+                            onChange={(e) => setSystemPrompt(e.target.value)}
+                            placeholder="Дополнительные инструкции для AI..."
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)